services/collectd/service.go

```diff
@@ -299,7 +299,12 @@ func (s *Service) UnmarshalCollectd(packet *gollectd.Packet) []models.Point {
 		p, err := models.NewPoint(name, tags, fields, timestamp)
 		// Drop invalid points
 		if err != nil {
-			s.Logger.Printf("Dropping point %v: %v", name, err)
```

Comment:
`name` == nil should not trigger here panic.  There was a panic in this code that was fixed via https://github.com/influxdata/influxdb/commit/01193668cf88658e3dc99a2f2274503b3cece989

Can you provide the stack trace and version of the server your are running?